src/test/java/org/zalando/nakadi/domain/BatchFactoryTest.java

```diff
@@ -132,4 +132,13 @@ public void testGarbageBetweenEvents() {
             fail();
         } catch (JSONException e) {}
     }
+
+    @Test
+    public void testNumberLargerThanMaxLongInEvents() {
+        final String events = "[{\"number\": 9223372036854775808 }]";
+        try {
+            BatchFactory.from(events);
+            fail();
+        } catch (JSONException e) {}
```

Comment:
@ferbncode 
You can do that in @Test annotation like `@Test(expected = JSONException.class)` instead of catching exception and calling `fail()`. 
Another thing that will improve - currently if `BatchFactory.from(events);` throws some other exception - your test will be green (but it should be red in such case)